src/packet.proto

```diff
@@ -36,3 +36,11 @@ message packet {
   routing_information routing = 9;
   window rx2_window = 10;
 }
+
+message verified_packet {
```

Comment:
Unless this is shared across services outside of oracles, which I don't think it is, move this to the packet verifier service?